lib/agenda.js

```diff
@@ -607,7 +600,7 @@ function processJobs(extraJob) {
     self._nextScanAt = new Date(now.valueOf() + self._processEvery);
     self._findAndLockNextJob(name, definitions[name], function(err, job) {
       if (err) {
-        throw err;
+        console.log('error with mongodb connection', err);
```

Comment:
Console.log probably isn't appropriate, and we don't know at this point that it's definitely a mongo error.. could have been passed any error